src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/RangeAttribute.cs

```diff
@@ -217,13 +217,13 @@ private void SetupConversion()
                     Func<object, object> conversion;
                     if (ConvertValueInInvariantCulture)
                     {
-                        conversion = value => value?.GetType() == type
```

Comment:
The conversion function is only invoked from `IsValid`, which already checks for null: https://github.com/dotnet/corefx/blob/015cf5d9832ed9d6734367c8a2e29dfa8bae4a25/src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/RangeAttribute.cs#L116-L138